attributes/default.rb

```diff
@@ -12,10 +12,10 @@
 default['pgd']['static_url'] = "/static/"	#Trailing slash is must
 
 # Settings.py database
-default['pgd']['database']['engine'] = "django.db.backends.mysql"
-default['pgd']['database']['name'] = "pgd-dev"
-default['pgd']['database']['user'] = "pgd-user"
-default['pgd']['database']['password'] = "pdg-pass"
+default['pgd']['database']['engine'] = nil
```

Comment:
As mentioned in IRC, the best way is probably something like [this](https://gist.github.com/jordane/14481ec4f5bbd9cbf60f) combined with a comment, and a note in the readme.